test/src/module/config/loadTest.c

```diff
@@ -267,6 +267,45 @@ testRun(void)
 
         unsetenv("PGBACKREST_REPO1_S3_KEY");
         unsetenv("PGBACKREST_REPO1_S3_KEY_SECRET");
+
```

Comment:
Add a test in loadTest.c to test for whether cfgOptionValid(cfgOptCompress) is true/false.

Reply:
Added several.